docker/ubuntu/Dockerfile

```diff
@@ -168,12 +198,34 @@ RUN /src/grass_build/configure \
   --with-bzlib \
   --with-pdal \
   --without-mysql \
-  --without-odbc \
-  --without-openmp \
+  --with-blas \
+  --with-lapack \
+  --with-readline \
+  --with-odbc \
+  --with-openmp \
   --without-opengl \
     && make -j $NUMTHREADS \
     && make install && ldconfig
 
+# Build the GDAL-GRASS plugin
+RUN git clone https://github.com/OSGeo/gdal-grass && \
+    cd "gdal-grass" && \
+    ./configure --with-gdal=/usr/bin/gdal-config \
+            --with-grass=/usr/local/grass84 && \
+    make -j $NUMTHREADS && \
+    make install -j $NUMTHREADS && \
+    cd /src && \
+    rm -rf "gdal-grass"
+
```

Comment:
Where is this compiled to? Is it also copied to the "grass" stage?

Reply:
> Where is this compiled to? Is it also copied to the "grass" stage?

Good catch! it actually goes to `/usr/lib/gdalplugins` and the GDAL_PLUGIN_PATH has to be set. Adressed in [cb8363b](https://github.com/OSGeo/grass/pull/3170/commits/cb8363b399f0252622169b0c18277898c4864c78) and included in the test script now...